Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM julia:1.6-alpine
```

Comment:
to jest zafixowane na 1.6 wersję Julia? To jest OK, ale się zastanawiam, czy 1.6 jest tylko dostępna, czy 1.7 może też być?

Reply:
są różne dostępne, zob. tu https://hub.docker.com/_/julia
na pewno musi być zafixowana wersja, żeby stabilnie działało i się nie rozjechało w przyszłości.
powodem dla 1.6 jest to, że jest większa niż 1.0 zgodnie z dokumentacją :)

czy chcemy inną wersję niż 1.6?

Reply:
sprawdziłam i działa
już zmienione w nowym commicie.